front/lib/production_checks/checks/scrub_deleted_core_document_versions.ts

```diff
@@ -1,19 +1,12 @@
-import type { Bucket, File } from "@google-cloud/storage";
-import { Storage } from "@google-cloud/storage";
-import { createHash } from "blake3";
+import { CoreAPI } from "@dust-tt/types";
 import type { LoggerOptions } from "pino";
 import type pino from "pino";
 
+import config from "@app/lib/api/config";
 import type { CheckFunction } from "@app/lib/production_checks/types";
-import {
-  getCorePrimaryDbConnection,
-  getCoreReplicaDbConnection,
-} from "@app/lib/production_checks/utils";
-import { withRetries } from "@app/lib/utils/retries";
-import logger from "@app/logger/logger";
+import { getCoreReplicaDbConnection } from "@app/lib/production_checks/utils";
 
-const { CORE_DATABASE_URI, SERVICE_ACCOUNT, DUST_DATA_SOURCES_BUCKET } =
-  process.env;
+const { CORE_DATABASE_URI } = process.env;
```

Comment:
Can we avoid using process.env?

Reply:
pre-existed and is meanted to be deleted as soon as it has been run a couple times and all scrubbing happens as part of the document deletion, won't fix.